packages/iov-ethereum/src/serialization.spec.ts

```diff
@@ -147,10 +153,12 @@ describe("Serialization", () => {
       {
         const tx: SendTransaction = {
           kind: "bcp/send",
-          creator: creator,
-          amount: amount,
-          gasPrice: gasPrice,
-          gasLimit: undefined,
+          creator,
+          amount,
+          fee: {
+            gasPrice,
+            gasLimit: undefined,
```

Comment:
I'm not really a fan of those shortcut notations where property name equals variable name. I find it very hard to parse in the case where some properties are key/value and some are variable only. Also it is a strange mix of concerns: the local variable name on the one hand, that should be changeable at any point in time and the external property name on the other hand that must not be changed.

So if you don't have a strong opinion here, I'd like to stick with explicit property names. We can also use tslint to enforce consistency.

Reply:
I like them because they can reduce reading quite a lot, but the most important thing for me is consistency. I suggest a separate issue to add a tslint rule.